src/libs/ajax.js

```diff
@@ -1120,7 +1120,7 @@ const AzureStorage = signal => ({
   },
 
   details: async (workspaceId = {}) => {
-    const res = await fetchWorkspaceManager(`workspaces/v1/${workspaceId}/resources?stewardship=CONTROLLED`,
+    const res = await fetchWorkspaceManager(`workspaces/v1/${workspaceId}/resources?stewardship=CONTROLLED&limit=1000`,
```

Comment:
Why 1000?

Reply:
No good reason other than it seemed a sufficiently large number.